docs/plugins.md

```diff
@@ -61,7 +68,7 @@ The first step is to create a custom Docker container that will handle the insta
 
 ```docker
 ARG PYTHON_VER
```

Comment:
We should be consistent if providing a default value for Nautobot version we should here as well.
```suggestion
ARG PYTHON_VER=3.9
```